tests/test_lib_caconnector.py

```diff
@@ -280,6 +281,25 @@ def test_05_templates(self):
         self.assertTrue(ddiff.days > 740, ddiff.days)
         self.assertTrue(ddiff.days < 760, ddiff.days)
 
+    def test_99_cleanup(self):
+        filelist = glob.glob("{0!s}/100*.pem".format(WORKINGDIR))
+        for f in filelist:
+            os.remove(f)
+
+        FILES = ["cornelius_user@localhost.localdomain_realm1.pem",
+                 "cornelius_user@localhost.localdomain_realm1.req",
```

Comment:
actually these two are not tracked and don't exist after a current run of the test.
So the `os.remove` call fails and the test breaks on a fresh checkout.

Reply:
will do

Reply:
done in 151a9d2d